testData/preferences/mary.json

```diff
@@ -4,10 +4,10 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
-                    "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
+                    "http://registry.gpii.net/common/onscreenKeyboard": true,
```

Comment:
I strongly prefer the name  `onScreenKeyboardEnabled` over `onscreenKeyboard` because it is more consistent with other common terms we've created so far; see http://wiki.gpii.net/w/Cloud4all_Third_Pilot:_Common_Terms.